src/shared/components/query/QueryStore.ts

```diff
@@ -825,7 +826,9 @@ export class QueryStore {
             let result: { [studyId: string]: string[] } = {};
 
             _.each(this.physicalStudiesSet.result, (study, studyId) => {
-                result[studyId] = [studyId];
+                if (isQueriedStudyAuthorized(study)) {
+                    result[studyId] = [studyId];
+                }
```

Comment:
I didn't add it because the unauthorized studies feature shouldn't apply to the virtual studies. Also in the StudyViewPageStore, I see that the isQueriedStudyAuthorized is added only to the physical studies.

Reply:
@alisman 
>it seems at least possible that an unauthorized study could make it's way into a virtual study from a user that has privelages to it.

This is already the case when not running with the unauthorized studies feature. Or am I missing something in your feedback?